packages/wrangler/src/config/validation.ts

```diff
@@ -2266,6 +2266,21 @@ const validateContainerAppConfig: ValidatorFn = (
 			);
 		}
 
+		if (
+			"rollout_step_percentage" in containerAppOptional &&
+			containerAppOptional.rollout_step_percentage !== undefined
+		) {
+			if (
+				typeof containerAppOptional.rollout_step_percentage !== "number" ||
+				containerAppOptional.rollout_step_percentage > 100 ||
+				containerAppOptional.rollout_step_percentage < 25
+			) {
+				diagnostics.errors.push(
+					'"containers.rollout_step_percentage" should be a number between 25 and 100'
```

Comment:
the format in which we surface these validation errors to users is smth like
```
<field> should be <validation_criteria>, but got <field_value>
```

Not blocking by any means, but just wanted to call it out, as I'm noticing other similar container-related errors are not adhering to that format either, so perhaps good to keep in mind in the future <3